src/app/routes.py

```diff
@@ -21,27 +28,43 @@ def dashboard():
     return render_template('dashboard.html')
 
 
-@main.route('/query', methods=['GET', 'POST'])
+@main.route('/query')
 @oidc.require_login
 def query():
-    auths = ['FOO', 'BAR', 'PUBLIC', 'PRIVATE']
+    cmd = 'authorization -c [PLACEHOLDER] -k [PLACEHOLDER]'.split(' ')
+    whoami = CliRunner().invoke(dwv_entry_point, cmd, standalone_mode=False).return_value
+    auths = whoami['proxied_users'][0]['auths']
+
+    return render_template('query.html', auths=auths)
+
 
+@main.route('/query_results', methods=['GET', 'POST'])
+@oidc.require_login
+def query_results():
     if request.method == 'POST':
-        query_name = request.form.get('query_name')
+        query_name = request.form.get('query_name') or uuid4()
         query_text = request.form.get('query')
         selected_auths = [auth for auth in request.form.getlist('auths') if auth]
         data_type = request.form.get('data_type')
-        output_location = request.form.get('output_location')
-        decode_raw_data = 'decode_raw_data' in request.form
-
-        return render_template('query_result.html', query_name=query_name,
-                               query_text=query_text,
-                               selected_auths=selected_auths,
-                               data_type=data_type,
-                               output_location=output_location,
-                               decode_raw_data=decode_raw_data)
+        decode_raw_data = bool(request.form.get('decode_raw_data'))
+
+        output_location = f'/tmp/{query_name}/results.json'
+        cmd = [
+            'query',
+            '-c', 'PLACEHOLDER', '-k', 'PLACEHOLDER',
+            '-q', query_text,
+            '--query-name', f'{query_name}',
+            '--auths', ",".join(selected_auths),
```

Comment:
Should add a way to guarantee they selected at least one auth

Reply:
![image](https://github.com/user-attachments/assets/94d1deed-82a0-4202-82d2-af9c6b892d7f)